src/dodal/devices/shutter.py

```diff
@@ -0,0 +1,44 @@
+from enum import Enum
+
+from bluesky.protocols import Movable
+from ophyd_async.core import AsyncStatus, StandardReadable, wait_for_value
+from ophyd_async.epics.signal import epics_signal_r, epics_signal_w
+
+
+class ShutterSetState(str, Enum):
+    CLOSED = "close"
+    OPEN = "open"
+    RESET = "reset"
+
+
+class ShutterState(str, Enum):
+    CLOSED = ShutterSetState.CLOSED
+    OPEN = ShutterSetState.OPEN
+    OPENING = "opening"
+    CLOSING = "closing"
+    FAULT = "fault"
+
+
+class Shutter(StandardReadable, Movable):
+    def __init__(self, prefix: str, name: str):
+        self.position_set = epics_signal_w(
+            write_pv=prefix + "CTRL2",
+            datatype=ShutterSetState,
+        )
+        with self.add_children_as_readables():
+            self.position_readback = epics_signal_r(
+                read_pv=prefix + "STA",
+                datatype=ShutterState,
+            )
+        super().__init__(name=name)
+
+    @AsyncStatus.wrap
+    async def set(self, position: ShutterSetState):
+        if position == ShutterSetState.RESET:
+            return
```

Comment:
@DominicOram why should this shortcut? What does set(Reset) do on the IOC?

Reply:
I don't know. To be honest, I'm not sure we should even expose it as an option but the Enum has to be exhaustive.